owncloudApp/src/main/java/com/owncloud/android/features/FeatureList.java

```diff
@@ -160,16 +161,24 @@ public Object createFromParcel(Parcel source) {
                 };
     }
 
-    static int versionCodeFromString(String version) {
+    private static int versionCodeFromString(String version) {
+        int versionCode = versionBetaCodeFromString(version);
+        if (versionCode == 0) {
+            Log_OC.w(TAG, "Version string is incorrect " + version);
+            return 0;
+        }
+
+        return versionCode;
+    }
+
+    private static int versionBetaCodeFromString(String version) {
```

Comment:
Why do you need this new method to remove the useless log entries? I would keep `versionCodeFromString` as it was before but adding just a new check `(versionCode == 0)` to log or not. Besides, you are not checking `versionCode == 0` for those direct calls to `versionBetaCodeFromString`.

Reply:
nop, with your suggestion I would ignore versionNumber=0, what's not correct
Current solution is a one check-code, but only warning for non-beta

Reply:
@hannesa2 see https://github.com/owncloud/android/pull/2443#discussion_r256507431